pom.xml

```diff
@@ -8,7 +8,7 @@
 
   <groupId>com.cloudbees.jenkins.plugins</groupId>
   <artifactId>docker-custom-build-environment</artifactId>
-  <name>CloudBees Docker Custom Build Environment Plugin</name>
+  <name>Docker Custom Build Environment Plugin. Maintained by CloudBees, Inc.</name>
```

Comment:
@oleg-nenashev Please, take a look to the other PRs related. This was what we decided.